components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/workflow/ApplicationUpdateApprovalWorkflowExecutor.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2005-2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This is not the correct license header. Check other places as well and add the correct one